docs/user/ppl/functions/datetime.rst

```diff
@@ -1906,11 +1854,11 @@ Description
 >>>>>>>>>>>
 
 Usage: Converts given argument to Unix time (seconds since Epoch - very beginning of year 1970). If no argument given, it returns the current Unix time.
-The date argument may be a DATE, DATETIME, or TIMESTAMP string, or a number in YYMMDD, YYMMDDhhmmss, YYYYMMDD, or YYYYMMDDhhmmss format. If the argument includes a time part, it may optionally include a fractional seconds part.
+The date argument may be a DATE, TIMESTAMP, or TIMESTAMP string, or a number in YYMMDD, YYMMDDhhmmss, YYYYMMDD, or YYYYMMDDhhmmss format. If the argument includes a time part, it may optionally include a fractional seconds part.
```

Comment:
Timestamp is included twice.

Reply:
removed in https://github.com/Bit-Quill/opensearch-project-sql/pull/336/commits/1aec4f9ee0ca2e447f3a4b9cdff2169a98c76fcb